src/lib.rs

```diff
@@ -66,6 +68,7 @@ pub fn load_and_index_pois(
     let es = args.es.clone();
     let langs = &args.langs;
     let rubber = &mut mimir::rubber::Rubber::new(&es);
+    let poi_dataset = format!("{}_poi_default", &args.dataset); // TODO: is `{}_poi_default` right?
```

Comment:
the current index name format used by mimir is `munin_<type>_<dataset>`, 
and there is potentially 2 different datasets have been imported by Fafnir, defined by `dataset` and  `dataset-nosearch` arguments.